lib/ui/painting/image_generator.h

```diff
@@ -80,7 +84,7 @@ class ImageGenerator {
   /// @return     Information about the given frame. If the image is
   ///             single-frame, a default result is returned.
   /// @see        `GetFrameCount`
-  virtual const FrameInfo GetFrameInfo(unsigned int frame_index) const = 0;
+  virtual const FrameInfo GetFrameInfo(unsigned int frame_index) = 0;
```

Comment:
Why remove the |const| modifier?

Reply:
`GetFrameInfo` requires decoding the image header as per the registry protocol, which is expected to have substantial side effects per the protocol.